enterprise/j2ee.ddloaders/src/org/netbeans/modules/j2ee/ddloaders/ejb/DDChangesPanel.java

```diff
@@ -101,12 +101,12 @@ public java.awt.Dimension getPreferredSize () {
         return new java.awt.Dimension(600, 400);
     }
     
-    public synchronized void setChanges (List changes) {
-        listModel.clear ();
+    public synchronized void setChanges (List<DefaultListModel> changes) {
```

Comment:
This method exists more or less identically also in:
- `org.netbeans.modules.j2ee.ddloaders.web.DDChangesPanel`
- `org.netbeans.modules.j2ee.ddloaders.client.DDChangesPanel`
- `org.netbeans.modules.j2ee.ddloaders.app.DDChangesPanel`
should these not be updated?

Reply:
Completely agreed.. They should also be changed..

Fixed..